### PR TITLE
add instructions for what to do if you add the wrong remote

### PR DESCRIPTION
**Pull request by retiform**
Something I managed to do just yesterday when  I had two repos for the same project in different places.

Just adds instructions on how to change the url, or how to delete a git project if you cloned the wrong thing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,6 +146,21 @@ To clone it into a folder with a different name than the default repository name
 $ git clone [url] name-of-new-folder
 ```
 
+### I set the wrong remote repository.
+
+There are a few possible problems here:
+
+
+If you cloned the wrong respistory simply delete the directory created after running git clone and clone the correct repository
```
Comment (RichardLitt) [inline README.md]: Put `git clone` in check marks.

File: README.md [hunk 2]
```diff
@@ -145,6 +146,21 @@ To clone it into a folder with a different name than the default repository name
 $ git clone [url] name-of-new-folder
 ```
 
+### I set the wrong remote repository.
+
+There are a few possible problems here:
+
+
```
Comment (RichardLitt) [inline README.md]: Limit to one empty space for lines, not two empty lines.

File: README.md [hunk 3]
```diff
@@ -145,6 +146,21 @@ To clone it into a folder with a different name than the default repository name
 $ git clone [url] name-of-new-folder
 ```
 
+### I set the wrong remote repository.
```
Comment (RichardLitt) [inline README.md]: No final period

File: README.md (hunk 1 above)
Comment (RichardLitt) [reply README.md]: Also, add a period.

File: README.md [hunk 4]
```diff
@@ -145,6 +146,21 @@ To clone it into a folder with a different name than the default repository name
...
+
+There are a few possible problems here:
+
+
+If you cloned the wrong respistory simply delete the directory created after running git clone and clone the correct repository
+
+
+If you set the wrong repository as the origin of an existing local repository, change the url of your origin by running
```
Comment (RichardLitt) [inline README.md]: Add a final colon.

File: README.md [hunk 5]
```diff
@@ -145,6 +146,21 @@ To clone it into a folder with a different name than the default repository name
...
+
+If you cloned the wrong respistory simply delete the directory created after running git clone and clone the correct repository
+
+
+If you set the wrong repository as the origin of an existing local repository, change the url of your origin by running 
+
+```sh
+$ git remote set-url origin [url of the actual repo]
+```
+For more see [this SO top](https://stackoverflow.com/questions/2432764/how-to-change-the-uri-url-for-a-remote-git-repository#2432799)
```
Comment (RichardLitt) [inline README.md]: Remove. Let's not link out.

Comment (retiform) [reply README.md]: The readme litterally states to link to SO and sources.

Comment (RichardLitt) [reply README.md]: Huh, I forgot that. :D Good catch. Keep this in.

**Comment by retiform** [comment]
Changes made seem fine.

**Comment by RichardLitt** [comment]
I think GitHub's 'resolve conversation' is a bit confusing. I was suggesting changes for you to make; you still need to go through and make them, instead of marking the conversations resolved. :)

File: README.md (hunk 3 above)
Comment (retiform) [reply README.md]: This conforms to every other header. Periods are not normally used in titles anyways.

File: README.md (hunk 1 above)
Comment (retiform) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (retiform) [reply README.md]: fixed

**Comment by RichardLitt** [comment]
Thanks. Merged!

**Pull request merged**